bundles/org.openhab.binding.mqtt.awtrixlight/README.md

```diff
@@ -0,0 +1,380 @@
+|-----------------------|--------------------------------------------------------------------------------------------------------------------------------------------|----------|----------|
+| `basetopic`           | The MQTT base topic for the Awtrix device                                                                                                  | Yes      | "awtrix" |
+| `appLockTimeout`      | Timeout in seconds before releasing the lock to a selected app and returning to normal app cycle (see App Configuration for more details). | No       | 10       |
+| `discoverDefaultApps` | Enable discovery of default apps. Since default apps cannot be controlled by this binding this should usually be disabled.                 | No       | false    |
+| `lowBatteryThreshold` | Battery level threshold for low battery warning.                                                                                           | No       | 25       |
+
+### App Configuration (`awtrixapp`)
+
+| Parameter    | Description                        | Required | Default |
+|--------------|------------------------------------|----------|---------|
+| `appname`    | Name of the app                    | Yes      | -       |
+| `useButtons` | Enable button control for this app | No       | false   |
+
+When you enable the button control for an app, you can lock the app to the display by pushing the select button on the clock device. A red indicator will be shown while the app is locked and will start to blink shortly before the lock ends. The lock will last for the appLockTimeout set for the bridge. As long as the app is locked the normal app cycle is disabled and you can control the app by pressing the left and right buttons or the select button on the clock device. Pressing a button while the app is locked will reset the lock timeout to the value set for appLockTimeout. Left and right button presses will emit button events on the clock itself and the selected app. The button events can be used by rules to change the displayed app or perform any other actions (for example change the text color of the app or skip the current song playing on your audio device).
+
+## Channels
+
+### Bridge Channels (`awtrixclock`)
+
+| Channel           | Type                 | Description                                                                                                                                    |
+|-------------------|----------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| `app`             | String               | Currently active app: Will show the name of the app that is currently shown on the display.                                                    |
+| `autoBrightness`  | Switch               | Automatic brightness control: The clock will adjust the display brightness automatically based on ambient light.                               |
```

Comment:
The `lower-case-hyphen` naming convention should be used for the channel id's

Reply:
Done